docs: adding Bearer Cloud integration doc (#1191)
Bearer/bearer: docs/guides/bearer-cloud.md

![Cloud dashboard](/assets/img/cloud-dashboard.jpg)

## Jira integration

The Jira integration is available on the *Settings > Integrations* page.

To use the integration, you must connect a Jira account and allow access to the required permissions through the OAuth login.

Following your company's best practices, you can provide access to an existing account or set up a new user in Jira specifically for this integration. Whichever option you choose, make sure the account has the access permissions required to create and update tickets in the projects you want to.

You have two ways to use the Jira Integration:
1. Creating a Jira Ticket directly from a finding.
![Create Jira Ticket](/assets/img/jira-integration/create.png)

2. Link a finding to an existing Jira ticket.
![Link Jira Ticket](/assets/img/jira-integration/link.png)

Once a finding is associated with a Jira ticket, you can quickly see it in the interface, view the ticket status and go to the ticket. 

![View Jira Ticket](/assets/img/jira-integration/view.png)

{% callout "warn" %}

> Findings on Bearer dashboard are marked resolved only when the associated code is fixed. For example, if the associated Jira ticket is closed, but no code fix has been applied, the finding will stay open on Bearer dashboard. The source of truth is always code.

Reviewer: ```suggestion
Findings on Bearer Cloud are only marked resolved when the associated code is fixed. If the associated Jira ticket is closed, but no code fix has been applied, the finding will stay open. The source of truth is always the code.
```

{% endcallout %}

## Slack integration

The Slack integration is available on the *Settings > Integrations* page.

> To use the integration, you must connect a Slack account and allow access to the required permissions through the OAuth login, then select a default channel where to receive notifications on new findings.

Reviewer: ```suggestion
To use the integration, you must connect a Slack account and allow access to the required permissions through the OAuth login, then select a default channel where you want to receive notifications on new findings.
```

> Here is below an example of a Slack notification triggered by a new finding:

Reviewer: ```suggestion
Below an example of a Slack notification triggered by a new finding:
```
